platform/ios/src/MGLMapView.h

```diff
@@ -723,6 +723,13 @@ MGL_EXPORT
  */
 @property(nonatomic) CGFloat decelerationRate;
 
+/**
+ A Boolean value indicating whether the rotation threshold should be increased
+ when zooming. By default, the threshold is 7.5 degrees. Setting this property
+ to `YES` increases the threshold to 25 degrees.
+ */
+@property(nonatomic) BOOL increasesRotationThresholdWhenZooming;
```

Comment:
1. Do we want this to be a public setting? The original issue is that it’s _slightly_ too easy to rotate while zooming, which is the behavior we should improve by default. By making this public API, we start to offer guarantees that I’m not sure we’re going to be happy with in the long term.
2. Whether or not this is public, we should put it on the gesture recognizer, not `MGLMapView`.